azurerm/modules/azurerm-adls/main.tf

```diff
@@ -9,3 +9,18 @@ resource "azurerm_storage_account" "storage_account_default" {
   account_replication_type = var.account_replication_type
   is_hns_enabled           = each.value.hns_enabled
 }
+
+
+resource "azurerm_storage_container" "storage_container_blob" {
+  for_each              = toset(var.container_blob)
+  name                  = each.value
+  storage_account_name  = azurerm_storage_account.storage_account_default["blob"].name
```

Comment:
`["blob"]` only makes sense, if a storage account is created with this label, which is might not be as it's passed as a variable.